cloudbuild-e2e-gae-standard.yaml

```diff
@@ -14,7 +14,7 @@
 
 steps:
   # Vendor dependencies, and zip the code.
-  - name: golang:1.23.1
+  - name: golang:1.23.8
```

Comment:
Currently the max supported version for GAE standard is `go1.23.6`.

Reply:
Any idea on the ETA? I imagine users are eagerly awaiting this

Reply:
I couldn't find this information publicly documented. The [runtime support schedule](https://cloud.google.com/appengine/docs/standard/lifecycle/support-schedule#go) does not mention anything about the patch versions.

Reply:
The ETA seems to be by the end of next week. 
We can wait to merge till then instead of changing the CI to ignore the GAE failure.

Reply:
The test `cloudbuild-e2e-gae-standard.yaml` is now passing. 

There are still some `not completed` workflows from previous go version (1.23.6 and 1.24.0). Are those necessary to be re-triggered or cleaned before merging ?

Reply:
Yes, I think they need to be cleaned up as those jobs do not exist anymore. 
Perhaps try pushing an empty commit.